python-client/giskard/models/huggingface/__init__.py

```diff
@@ -1,15 +1,17 @@
 from typing import Union
 import logging
-
 from scipy import special
-import torch
 from pathlib import Path
 import yaml
-
 from giskard.core.core import SupportedModelTypes
 from giskard.models.base import WrapperModel
 
-from transformers import pipeline, pipelines
+try:
```

Comment:
can be removed since this case is already handled in the ai-test-v2-merged branch